library/build.gradle

```diff
@@ -2,8 +2,8 @@ apply plugin: 'com.android.library'
 
 dependencies {
     compile 'com.klinkerapps:logger:1.0.3'
-    compile 'com.squareup.okhttp:okhttp:2.5.0'
-    compile 'com.squareup.okhttp:okhttp-urlconnection:2.5.0'
+    compile 'com.squareup.okhttp3:okhttp:3.8.1'
+    compile 'com.squareup.okhttp3:okhttp-urlconnection:3.0.0'
```

Comment:
This needs to be the save 3.8.1 as okhttp, it was causing build errors in the sample app:

```groovy
compile 'com.squareup.okhttp3:okhttp-urlconnection:3.8.1'
```

Any reason why you didn't have it as 3.8.1?